pyartcd/pyartcd/slack.py

```diff
@@ -42,15 +42,26 @@ def bind_channel(self, channel_or_release: Optional[str]):
         else:
             raise ValueError(f"Invalid channel_or_release value: {channel_or_release}")
 
-    async def say_in_thread(self, message: str, reaction: Optional[str] = None):
+    async def get_channel_id(self):
+        """
+        Get ID of the currently bound channel
+        """
+        # Can only get private channel ids for now to avoid longer waiting caused by request ratelimit
+        response = await self._client.conversations_list(types="private_channel", limit=999)
```

Comment:
@adobes1 I thought we discussed that we can get the channel ID directly from the message response, when we are posting to the channel?

Querying all channels from RH Slack is not efficient. PTAL and fix this once you're back. Reverting PR.